cds-snc/platform-forms-client
# Feat: Campaign Banner as a feature flag

Closes #4092

This PR converts the Campaign Banner into a Feature Flag. 

Testing involves a few steps:
1. A fresh env should have no banner.
2. Admin!~ Feature Flags -> "Campaign" should be available.
3. Then also Campaign Text needs to be set:

```
"COMING SOON",
"<strong>In August</strong>: Build multi-page forms with branching logic. <a href='/en/contact' className='text-white hover:text-white'>Subscribe for updates</a>"
```

```
"À VENIR"
"<strong>En août</strong> : Créez des formulaires multipages avec une logique d’embranchement. <a href='/fr/contact' className='text-white hover:text-white'>S’abonner aux mises à jour</a>"
```
4. Check out the banner!~
5. Go back, edit the Campaign Text, note the changes. 
6. Turn off feature flag. No banner should appear.

Voila no dev required!~

"NEW" Banners are:
```
"NEW",
"Build complex multi-page forms with display logic and branching. <a href='https://articles.alpha.canada.ca/forms-formulaires/setting-up-form-logic/' className='text-white hover:text-white'>Learn more</a>"
```

```
"NOUVEAUTÉ"
"Créez des formulaires complexes de plusieurs pages avec logique d’affichage et d’embranchement. <a href='https://articles.alpha.canada.ca/forms-formulaires/fr/configurer-formulaire-avec-logique/' className='text-white hover:text-white'>En savoir plus</a>"
```
